WordPress/Classes/Utility/Logging/CrashLogging+Singleton.swift

```diff
@@ -1,8 +1,11 @@
 import Foundation
 import AutomatticTracks
 
-extension CrashLogging {
+extension WPLoggingStack {
+    static let shared = WPLoggingStack()
```

Comment:
We shouldn't use using `WordPressAppDelegate.shared` as it's main-thread confined.